fontique/src/generic.rs

```diff
@@ -214,7 +214,7 @@ mod tests {
         let mut value = 0;
         while value <= GenericFamily::MAX_VALUE {
             // Safety: In a const context, therefore if this makes an invalid GenericFamily, that will be detected.
-            // TODO: use `&raw const value` instead of the addr_of!
+            // When updating the MSRV to 1.82 or later, this can use `&raw const value` instead of the addr_of!
```

Comment:
Once this lands, I'll submit another PR that just fixes this and removes the comment.

Reply:
This was fixed in PR #308 separately.